src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/AttachExceptionRecordToExistingCaseTest.kt

```diff
@@ -1,87 +1,207 @@
 import io.restassured.RestAssured
 import io.restassured.RestAssured.given
 import io.restassured.builder.RequestSpecBuilder
 import io.restassured.http.ContentType.JSON
-import org.hamcrest.Matchers.contains
+import io.restassured.response.Response
+import io.restassured.response.ValidatableResponse
+import io.restassured.response.ValidatableResponseOptions
+import io.restassured.specification.RequestSpecification
+import org.apache.http.HttpHeaders.AUTHORIZATION
 import org.hamcrest.Matchers.equalTo
+import org.hamcrest.Matchers.hasItem
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.beans.factory.annotation.Value
 import org.springframework.boot.web.server.LocalServerPort
 import org.springframework.test.context.junit.jupiter.SpringExtension
 import u.gov.hmcts.reform.bulkscan.orchestrator.controllers.config.PortWaiter.waitFor
+import uk.gov.hmcts.reform.bulkscan.orchestrator.controllers.config.Environment
+import uk.gov.hmcts.reform.bulkscan.orchestrator.controllers.config.Environment.CASE_REF
+import uk.gov.hmcts.reform.bulkscan.orchestrator.controllers.config.Environment.JURIDICTION
 import uk.gov.hmcts.reform.bulkscan.orchestrator.controllers.config.IntegrationTest
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.CallbackValidations
 import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest
-import uk.gov.hmcts.reform.ccd.client.model.CallbackTypes
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest.CallbackRequestBuilder
 import uk.gov.hmcts.reform.ccd.client.model.CaseDetails
+import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse
+
+typealias ResponseValidation = ValidatableResponseOptions<ValidatableResponse, Response>
+
+fun RequestSpecification.postToCallback(type: String = "attach_case") = post("/callback/{type}", type)
+fun RequestSpecification.setBody(builder: CallbackRequestBuilder) = body(builder.build())
+fun ResponseValidation.shouldContainError(error: String) = body("errors", hasItem(error))
+
+fun MappingBuilder.authorised() = with(this) {
+    withHeader(AUTHORIZATION, containing("eyJhbGciOiJIUzI1NiJ9."))
+    //TODO cant seem to make this match
+//    withHeader("ServiceAuthorization", containing("eyJhbGciOiJIUzI1NiJ9."))
+}
 
 @ExtendWith(SpringExtension::class)
 @IntegrationTest
 class AttachExceptionRecordToExistingCaseTest {
+    companion object {
+        private val mapper: ObjectMapper = ObjectMapper()
+        fun asJson(obj: Any): String = mapper.writeValueAsString(obj)
+    }
 
     @LocalServerPort
     private var applicationPort: Int = 0
 
+    @Value("\${wiremock.port}")
+    private var wireMockPort: Int = 0
+
+    private val wireMock by lazy { WireMock(wireMockPort) }
+
+    private val startEvent = get(
+        "/caseworkers/640/jurisdictions/BULKSCAN/case-types/Bulk_Scanned"
+            + "/cases/1539007368674134/event-triggers/attachScannedDocs/token"
```

Comment:
can constants be used? especially for `1539007368674134` as it's used later in tests